### DISCUSSION TITLE
中央程式相關修課的程式碼 Repo

### DISCUSSION THREAD

**Discussion by 1chooo** [1101-GS4719-C/Python-Community-GS4719-C#1867] [General]
大家好，這個 repo 是我在中央的修課紀錄，裡面放了一些我當時的筆記，當時為了完成修課所寫的程式碼，整理方式是用我修的時候大幾去編排，之後會繼續補上更多課堂相關資訊以及課堂的相關資訊，希望可以幫助到大家，如果有相同課號，並且同學也有修習過，也可以提供給上這個 Repo，我會負責整理進去！大家期末順利！

- repo 連結: https://github.com/1chooo/my-uni-courses

（如有涉及到不可公開資訊可以直接發 issue 給我！一定馬上修改）

**Comment by chia-kai-chang** [comment]
大家星星刷起來~ ⭐⭐⭐⭐⭐⭐

**Reply by 1chooo** [reply]
謝謝老師支持！！！！

**Comment by benbebobo149** [comment]
我覺得你的想法很棒，我也想像你一樣做出整理課程的repo，但我擔心說會觸碰到版權，現在感覺抓得超嚴，不太適合直接把老師的講課內容放上去。或許我們可以考慮分享自己的學習筆記或是專題成果，這樣既可以避免侵犯版權，又能讓其他人受益。另外，我也贊同將知識轉化為自己的理解和心得，這樣更有助於加深自己對知識的理解和記憶。最後，我要感謝同學分享這個激發我思考的好點子。

**Comment by lyykey** [comment]
推個，我在大學修課很多前提是這堂課是否有先人的智慧，不論是課程的講義、作業還是上課錄影，其實很多時候都是為我提供課前的預習以及對於不懂的地方可以再次學習的資源，感謝分享。

**Comment by YifenLin** [comment]
我自己最近的感慨是，大學是一個學習一樣技能CP值最高的地方，像是在外面補習或是購買課程至少就要500/hr。其實我自己是希望在大學的時候多修一點課的，但是我又不得不向現實低頭，畢竟推甄看得終究是成績。感謝分享資源，隨時有時間想要充實自己都可以!
